modules/migrations/base/pullrequest.go

```diff
@@ -13,7 +13,6 @@ import (
 // PullRequest defines a standard pull request information
 type PullRequest struct {
 	Number         int64
-	OriginalNumber int64 `yaml:"original_number"`
```

Comment:
This may breaks something on migrating repository into files. 🤔

Reply:
That field was only used by Gitlab and should be 0 in all dumped files.